code/lib/core-common/src/utils/validate-configuration-files.ts

```diff
@@ -1,20 +1,29 @@
 import { dedent } from 'ts-dedent';
 import glob from 'glob';
 import path from 'path';
+import { readConfig } from '@storybook/csf-tools';
+import { once } from '@storybook/node-logger';
 
 import { boost } from './interpret-files';
 
-export function validateConfigurationFiles(configDir: string) {
+export async function validateConfigurationFiles(configDir: string) {
   const extensionsPattern = `{${Array.from(boost).join(',')}}`;
-  const exists = (file: string) =>
-    !!glob.sync(path.resolve(configDir, `${file}${extensionsPattern}`)).length;
+  const mainConfigMatches = glob.sync(path.resolve(configDir, `main${extensionsPattern}`));
```

Comment:
Can we turn this into a async call, now that this function is async?

Reply:
Done!